front/app/modules/commercial/native_surveys/admin/components/SurveyBuilderTopBar/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+// hooks
+import useProject from 'hooks/useProject';
+import useLocalize from 'hooks/useLocalize';
+
+// components
+import GoBackButton from 'components/UI/GoBackButton';
+import Button from 'components/UI/Button';
+
+// styling
+import { colors } from 'utils/styleUtils';
+import {
+  Box,
+  stylingConsts,
+  Spinner,
+  Text,
+  Title,
+} from '@citizenlab/cl2-component-library';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// routing
+import clHistory from 'utils/cl-router/history';
+import { withRouter, WithRouterProps } from 'utils/cl-router/withRouter';
```

Comment:
Is the withRouter/WithRouterProps deprecated now? We have a new hook somewhere I believe that we can use instead.